packages/liveblocks-react/src/room.tsx

```diff
@@ -1983,10 +2011,6 @@ export function createRoomContext<
     React.useEffect(() => {
       const queryKey = makeNotificationSettingsQueryKey(room.id);
       void getInboxNotificationSettings(room, queryKey);
-
-      incrementQuerySubscribers(queryKey);
```

Comment:
We used to poll room notification settings for the user every 5 minutes (like we did for threads and notifications) but I am curious why we do that? 

@marcbouchenoire In an upcoming PR, I want to increase/decrease polling interval for threads/notifications based on the room connection status, etc, and I am guessing we wouldn't want the same for polling room notification setting if we even do in the first place.

Reply:
In this PR, I have removed polling room notification settings, but I am happy to introduce it back if you think there is a need for that

Reply:
I haven't worked on this but I assume it's because there's no WS event for this, so without polling you could have infinitely stale scenarios (e.g. you changed a notification settings dropdown to "none" in one tab and another tab with the same room never gets this change until a reload)